### PR TITLE
Code supplement

### PR DESCRIPTION
**Pull request by huangKai-2323**
### What is the purpose of the change

fix  missing code

### Related issues/PRs

Related issues: #590
Related pr:#591

### Brief change log

- Define the core abstraction and interfaces of the EngineConn Factory;
- Define the core abstraction and interfaces of Executor Manager.

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

File: linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/computation/client/FlinkOnceJobTest.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
 public class FlinkOnceJobTest {
   public static void main(String[] args) {
     // TODO First, set the right gateway url.
-    LinkisJobClient.config().setDefaultServerUrl("http://127.0.0.1:9002");
+    LinkisJobClient.config().setDefaultServerUrl("http://10.107.118.104:9001");
```
Comment (peacewong) [inline linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/computation/client/FlinkOnceJobTest.java:30]: use http://127.0.0.1:9002

File: linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/computation/client/SeatunnelOnceJobTest.java [hunk 2]
```diff
@@ -23,7 +23,7 @@
 
 public class SeatunnelOnceJobTest {
   public static void main(String[] args) {
-    LinkisJobClient.config().setDefaultServerUrl("http://ip:9001");
+    LinkisJobClient.config().setDefaultServerUrl("http://10.107.97.166:9001");
```
Comment (peacewong) [inline linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/computation/client/SeatunnelOnceJobTest.java:26]: use http://127.0.0.1:9001

File: linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/computation/client/SparkOnceJobTest.java [hunk 3]
```diff
@@ -31,7 +31,7 @@ public class SparkOnceJobTest {
 
   public static void main(String[] args) throws InterruptedException {
 
-    LinkisJobClient.config().setDefaultServerUrl("http://127.0.0.1:9001");
+    LinkisJobClient.config().setDefaultServerUrl("http://10.107.118.104:9001");
```
Comment (peacewong) [inline linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/computation/client/SparkOnceJobTest.java:34]: http://127.0.0.1:9001

Comment (huangKai-2323) [reply linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/computation/client/SparkOnceJobTest.java]: done

File: linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/computation/client/SeatunnelOnceJobTest.java (hunk 2 above)
Comment (huangKai-2323) [reply linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/computation/client/SeatunnelOnceJobTest.java]: done

File: linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/computation/client/FlinkOnceJobTest.java (hunk 1 above)
Comment (huangKai-2323) [reply linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/computation/client/FlinkOnceJobTest.java]: done

**Pull request merged**